actioncable/lib/action_cable/server/configuration.rb

```diff
@@ -7,7 +7,7 @@ class Configuration
       attr_accessor :connection_class, :worker_pool_size
       attr_accessor :channel_load_paths
       attr_accessor :disable_request_forgery_protection, :allowed_request_origins
-      attr_accessor :cable, :url
```

Comment:
@rafaelfranca `mount_path` is where to put it; `url` is where to expect to find it (if mount_path is nil).

In the separate-ACa-server use case, `mount_path` is nil, and `url` is `wss://cables.example.com/`.

In the default case, `mount_path` is `/cable`, and `url` is (nil, which implies) `/cable`.